src/slurm_plugin/jobinfomgtd.py

```diff
@@ -0,0 +1,184 @@
+# Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the
+# License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import logging
+import os
+from configparser import ConfigParser
+from datetime import datetime, timezone
+from logging.config import fileConfig
+
+from botocore.config import Config
+from common.schedulers.slurm_commands import get_jobs_info
+from common.time_utils import seconds
+from common.utils import sleep_remaining_loop_time
+from retrying import retry
+from slurm_plugin.cluster_event_publisher import ClusterEventPublisher
+from slurm_plugin.common import log_exception
+
+CONFIG_FILE_DIR = "/etc/parallelcluster"
+LOOP_TIME = 180
+log = logging.getLogger(__name__)
+event_logger = log.getChild("events")
+
+
+# Utils
+def _seconds(sec):
+    """Convert seconds to milliseconds."""
+    return sec * 1000
+
+
+def _minutes(min):
+    """Convert minutes to seconds."""
+    return min * 60
+
+
+def _write_json_to_file(filename, json_data):
+    """Write json to file."""
+    with open(filename, "w", encoding="utf-8") as file:
+        file.write(json.dumps(json_data))
```

Comment:
Are these being used? I don't see them called from anywhere?

Reply:
No, it was used for testing before I get the event publisher, now I can remove it